core/controllers/classroom.py

```diff
@@ -156,21 +156,11 @@ def get(self) -> None:
         classrooms = classroom_config_services.get_all_classrooms()
 
         for classroom in classrooms:
-            classroom_id_index_mapping_dict: Dict[str, str|int]
-            # TODO(#20845): Remove this custom logic once we have
-            # populated the index field of the math classroom.
-            if classroom.index is not None:
-                classroom_id_index_mapping_dict = {
-                    'classroom_id': classroom.classroom_id,
-                    'classroom_name': classroom.name,
-                    'classroom_index': classroom.index
-                }
-            else:
-                classroom_id_index_mapping_dict = {
-                    'classroom_id': classroom.classroom_id,
-                    'classroom_name': classroom.name,
-                    'classroom_index': 0
-                }
+            classroom_id_index_mapping_dict: Dict[str, str|int] = {
```

Comment:
@AFZL210 Could you please make the typing for this dict more specific, e.g. with TypedDict? I.e. the keys should be set in stone and the values for each key should have specific types.

Reply:
done